app/javascript/packs/application.tsx

```diff
@@ -47,7 +47,7 @@ initReact({
   'mentoring-queue': (data: any) => (
     <Mentoring.Queue request={data.request} sortOptions={data.sort_options} />
   ),
-  'student-tracks-list': (data) => (
+  'student-tracks-list': (data: any) => (
```

Comment:
At some point we could consider adding a type instead of `any`. We can do that later though.

Reply:
Absolutely! I only did this to make it work with `strict: true`!